<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Watch_cards</class>
 <widget class="QDialog" name="Watch_cards">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>413</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Watch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="watchWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0,0,0">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(43, 107, 111, 1);
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="styleSheet">
            <string notr="true">#label_18{
color: #FFFFFF;
text-align: center;
font-family: Work Sans;
font-size: 16px;
font-style: normal;
font-weight: 600;
line-height: normal;
}</string>
           </property>
           <property name="text">
            <string>Watch</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="watch">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditWatchSearch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">   background-image: url(:/search.png);
    background-repeat: no-repeat;
    background-position: right center;
	background-origin: content;
    padding-right: 20px; 
    padding-left: 6px;
    padding-top: 2px;
    padding-bottom: 2px;
    border: 1px solid #C2CCC5;
    color: #3D3D3D;
    font-size: 12px;
    border-radius: 14px; /* Adjust this value for your desired border radius */

</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetWatch">
        <property name="styleSheet">
         <string notr="true">#listWidgetWatch{
background: #F3FEFF;
border-bottom: 21px solid gray; /* Adjust border width for desired shadow size */

  border-bottom-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, /* Gradient from top to bottom */
                                    stop: 0 rgba(0, 0, 0, 0), /* Black with some transparency at top */
                                    stop: 1 rgba(0, 0, 0,.2) /* Fades to transparent at bottom */
                                   );
}
#listWidgetWatch::item:selected { 
background-color: #A1E3E7; 
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="movement">
         <enum>QListView::Free</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="watchListSelectedDetails_Parent" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_17{
	border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-right-radius: 7px;
    border-bottom-left-radius: 7px;
	background: #F7FCFC;
	backdrop-filter: blur(4.5px);
	background-color: rgb(255, 255, 255);


}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="Index_Name">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="watch_val1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="watch_indicator">
             <property name="minimumSize">
              <size>
               <width>6</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>6</width>
               <height>10</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="watch_val2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="Buy_Watch_Button">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#Add_Watch_Button{
border-radius: 2px;
background: #197CB1;
color: #FFF;
font-size: 12px;
font-style: normal;
font-weight: 700;
line-height: normal;
}

#Add_Watch_Button:hover {
    background: #156693; /* Change color on hover */
}

#Add_Watch_Button:disabled {
    background: #CCC; /* Change color when disabled */
    color: #666; /* Change text color when disabled */
}</string>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Sell_Watch_Button">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#Subtract_Watch_Button{
border-radius: 2px;
background: #CC3437;
color: #FFF;
font-size: 12px;
font-style: normal;
font-weight: 700;
line-height: normal;
}
#Subtract_Watch_Button:hover {
    background: #B92E31; /* Change color on hover */
}

#Subtract_Watch_Button:disabled {
    background: #CCC; /* Change color when disabled */
    color: #666; /* Change text color when disabled */
}</string>
             </property>
             <property name="text">
              <string>S</string>
             </property>
             <property name="icon">
              <iconset theme="edit-delete">
               <normaloff>../</normaloff>../</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="horizontalLayout_16" native="true">
           <layout class="QHBoxLayout" name="watchListSelectedDetailsParent">
            <property name="spacing">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_24">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="Open_Label">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #848484;</string>
                </property>
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Open_Num">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_27">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="High_Label">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #848484;</string>
                </property>
                <property name="text">
                 <string>High</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="High_Num">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_30">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="Low_Label">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #848484;</string>
                </property>
                <property name="text">
                 <string>Low</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Low_Num">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_28">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="Volume_Label">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #848484;</string>
                </property>
                <property name="text">
                 <string>Volume</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Volume_Num">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
